src/commands/show.ts

```diff
@@ -49,7 +50,34 @@ async function showProblemInternal(id: string): Promise<void> {
             return;
         }
 
-        const outDir: string = await selectWorkspaceFolder();
+        let outDir: string = await selectWorkspaceFolder();
+        const outputPath: string = leetCodeConfig.get<string>("outputPath") || "";
+        const problem: IProblem | undefined = listProblems.find((item: IProblem) => item.id === id);
+        switch (outputPath) {
```

Comment:
Steps to treat with the `outputPath`:

## Step 1: test if it's braced by `${}`:
> You can do this using regex: `\$\{(.*?)\}`, the regex can help you extract the value inside the braces

If yes, translate for `tag`, `language`, `difficulty`

### Why?
1. If the user provides a value which is not supported yet, for example: `${company}`, we will have a chance to tell the user this category is not supported yet
2. Use `switch(value.toLocaleLowerCase())` to make sure if the user provides `${Difficulty}`, the feature will still work.

## Step 2: join the path
If it's not braced by `${}`, then we can directly join the path without translation.

There is no need to treat with empty value because:

1. `leetCodeConfig.get<string>("outputPath") || ""` make sure the value will not be undefined
2. `path.join("a", "")` will ignore the empty string